### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Sinuousity**
Same as last closed pull request... except this one works.

### CODE REVIEW

File: Assets/Scripts/TES/MorrowindEngine.cs [hunk 1]
```diff
@@ -301,20 +302,23 @@ public void CastInteractRay()
 					GenericObjectComponent component = hitObj.gameObject.GetComponentInParent<GenericObjectComponent>();
 					if ( component != null )
 					{
-						switch ( component.gameObject.tag )
+						if ( !string.IsNullOrEmpty( component.objData.name ) )
```
Comment (ColeDeanShepherd) [inline Assets/Scripts/TES/MorrowindEngine.cs:305]: Wow, somehow I never knew IsNullOrEmpty existed...

File: Assets/Scripts/TES/MorrowindEngine.cs [hunk 2]
```diff
@@ -551,9 +556,10 @@ private GameObject InstantiateLight(LIGHRecord LIGH, bool indoors)
 			lightComponent.range = 3 * (LIGH.LHDT.radius / Convert.meterInMWUnits);
 			lightComponent.color = new Color32(LIGH.LHDT.red, LIGH.LHDT.green, LIGH.LHDT.blue, 255);
 			lightComponent.intensity = 1.5f;
+			lightComponent.bounceIntensity = 0f;
 			//lightComponent.shadows = LightShadows.Soft;
 
-			if(!indoors)
+			if(!indoors & false)//disabling exterior cell lights because there is no day/night cycle
```
Comment (ColeDeanShepherd) [inline Assets/Scripts/TES/MorrowindEngine.cs]: I'm confused by this line: doesn't the condition always evaluate to false, resulting in lights never being disabled?

**Comment by ColeDeanShepherd** [comment]
Everything looks good aside from the "if(!indoors & false)" line.

File: Assets/Scripts/TES/MorrowindEngine.cs (hunk 2 above)
Comment (Sinuousity) [inline Assets/Scripts/TES/MorrowindEngine.cs]: That's correct. I was using this temporarily but I've already added the option to the local settings so you can can toggle it from there instead.

File: Assets/Scripts/TES/MorrowindEngine.cs (hunk 1 above)
Comment (Sinuousity) [inline Assets/Scripts/TES/MorrowindEngine.cs:305]: I was having problems with object interaction where objects had white space names. Very handy!

**Comment by Sinuousity** [comment]
Wondering if I should close this request (again) and put in a new one with the new fixes.

**Pull request merged**